rootex/assets/materials/sprite.basic.rmat

```diff
@@ -0,0 +1,4 @@
+{
+    "diffuse": "rootex/assets/rootex.png",
+    "isLit": 0
+}
```

Comment:
sprite should be inheriting from ModelComponent, doesn't make sense to create a custom format for it